syntax = "proto3";

package cakelab.arflow_grpc.v1;

import "cakelab/arflow_grpc/v1/vector2.proto";
import "cakelab/arflow_grpc/v1/vector2_int.proto";

option csharp_namespace = "CakeLab.ARFlow.Grpc.V1";

/// https://docs.unity3d.com/Packages/com.unity.xr.arfoundation@6.0/api/UnityEngine.XR.ARSubsystems.XRCameraIntrinsics.html
message Intrinsics {
  Vector2 focal_length = 1;
  Vector2 principal_point = 2;
  Vector2Int resolution = 3;
}
